easybuild/easyconfigs/l/LHAPDF/LHAPDF-6.5.4-GCC-12.3.0.eb

```
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'LHAPDF'
+version = '6.5.4'
+
+homepage = 'http://lhapdf.hepforge.org/'
+description = """
+Les Houches Parton Density Function
+
+LHAPDF is the standard tool for evaluating parton distribution functions (PDFs) in high-energy physics.
+"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://www.hepforge.org/archive/lhapdf/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['2443a4b32cc3b0597c8248bd6e25703ace9c91a7a253c5f60b1b5428ef9c869e']
+
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+configopts = "--disable-static --enable-python"
+
+sanity_check_paths = {
+    'files': ["bin/lhapdf", "bin/lhapdf-config", "lib/libLHAPDF.%s" % SHLIB_EXT],
+    'dirs': ["share/LHAPDF"],
+}
+
+sanity_check_commands = [
+    "lhapdf --version",
+    "lhapdf-config --version",
+    "python -c 'import lhapdf'",
+]
+
+modloadmsg = """
+To use/install your own PDF data you need to adjust the path `export LHAPDF_DATA_PATH=/local/.../path/`.
+Then you can run `lhapdf update` followed by `lhapdf install your_pdf`.
+"""
```

Is there a particular reason for changing the text compared to the existing `[LHAPDF-6.5.3-GCC-11.3.0.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/LHAPDF/LHAPDF-6.5.3-GCC-11.3.0.eb)`?

```suggestion
modloadmsg = """
To use/install your own PDF data, you need to adjust the $LHAPDF_DATA_PATH:
`export LHAPDF_DATA_PATH=/local/.../path/:$LHAPDF_DATA_PATH`
Then you can run `lhapdf update` followed by `lhapdf install your_pdf`.
"""
```

---

Thanks. I think it has been changed in my 6.5.3 pull request and I have not adjusted my previous `.eb`.

---

ah, that makes sense :+1: